src/plugins/invisibleChat/components/EncryptionModal.tsx

```diff
@@ -85,7 +85,7 @@ function EncModal(props: ModalProps) {
                         const toSend = noCover ? encrypted.replaceAll("d", "") : encrypted;
                         if (!toSend) return;
 
-                        insertTextIntoChatInputBox(toSend);
+                        findByProps("editMessage").sendMessage(SelectedChannelStore.getChannelId(), { content: toSend });
```

Comment:
this causes errors because your message body is missing required fields

Reply:
I just added sendMessage in @utils/discord, please use it:
```suggestion
                        sendMessage(SelectedChannelStore.getChannelId(), { content: toSend });
```